### PR TITLE
DBZ-1178 Synchronize Debezium images

### PR DESCRIPTION
**Pull request by jpechane**
https://issues.jboss.org/browse/DBZ-1178

### CODE REVIEW

**Comment by gunnarmorling** [comment]
@jpechane, I'd expect this requires any doc updates, too? It should be mentioned somewhere how to reference those images, i.e. with the registry name as a prefix.

**Comment by jpechane** [comment]
@gunnarmorling intallation link redirected to quay,io. There every image contains correct `docker pull` command

File: documentation/modules/ROOT/pages/install.adoc [hunk 1]
```diff
@@ -35,7 +35,7 @@ The connector plugins are available from Maven:
...
 * https://repo1.maven.org/maven2/io/debezium/debezium-connector-cassandra/{install-version}/debezium-connector-cassandra-{install-version}-plugin.tar.gz[Cassandra plugin archive] (incubating)
 
-If immutable containers are your thing, then check out https://hub.docker.com/r/debezium/[Debezium's Docker images] for Zookeeper, Kafka, and Kafka Connect with the MySQL and MongoDB connectors already pre-installed and ready to go. Our xref:tutorial.adoc[tutorial] even walks you through using these images, and this is a great way to learn what Debezium is all about. You can even xref:operations/openshift.adoc[run Debezium on OpenShift].
+If immutable containers are your thing, then check out https://quay.io/organization/debezium[Debezium's Docker images] (https://hub.docker.com/r/debezium/[alternative]) for Zookeeper, Kafka, and Kafka Connect with the MySQL and MongoDB connectors already pre-installed and ready to go. Our xref:tutorial.adoc[tutorial] even walks you through using these images, and this is a great way to learn what Debezium is all about. You can even xref:operations/openshift.adoc[run Debezium on OpenShift].
```
Comment (gunnarmorling) [inline documentation/modules/ROOT/pages/install.adoc]:
```suggestion
If immutable containers are your thing, then check out https://quay.io/organization/debezium[Debezium's container images] (https://hub.docker.com/r/debezium/[alternative source] on DockerHub) for Apache Kafka, Kafka Connect and Apache Zookeeper, with the different Debezium connectors already pre-installed and ready to go. Our xref:tutorial.adoc[tutorial] even walks you through using these images, and this is a great way to learn what Debezium is all about.
Of course you also can run Debezium on Kubernetes and xref:operations/openshift.adoc[OpenShift].
```

**Comment by jpechane** [comment]
@gunnarmorling I'd like to take gradual approach. If possible your last PR for examples should use quay.io images.
Then when proved to be wokring we should convert the rest of examples. And the tutorial should be the last one as it is public face and we need to be absolutely sure that all works as it should.

**Comment by gunnarmorling** [comment]
Deal!

**Pull request merged**